imap/http_jmap.c

```diff
@@ -625,6 +625,36 @@ static int jmap_download(struct transaction_t *txn)
     return res;
 }
 
+static int sharedrights_cb(const mbentry_t *mbentry, void *vrock)
```

Comment:
Nit: Rename this to has_shared_rw_rights_cb ? This function really just that boolean question, and it's less confusing to than having the same named function to different things in two files.